src/styles/blocks/main-page.scss

```diff
@@ -0,0 +1,26 @@
+/* #region main-page*/
+
+.main {
+
+  &__container {
```

Comment:
apply requirement for your catalog

there should always be 4 cards in a row (not 2, 3 or 5)